samples_project/Assets/StarterAssets/Editor/PackageChecker/StarterAssetsPackageChecker.dll.meta

```diff
@@ -0,0 +1,33 @@
+fileFormatVersion: 2
```

Comment:
this .dll.meta is comitted but there is no .dll committed.

Reply:
> Your PR is also missing a project dependency on Cinemachine which is required.
> 
> In short, I think we can accomplish what this PR is trying to show without bringing all of this in.

so you mean to still use this starter asset but select those scripts that are used?